.rubocop.yml

```diff
@@ -7,19 +7,11 @@ AllCops:
   TargetRubyVersion: 2.7
   TargetRailsVersion: 5.2
   Exclude:
-    - 'config/initializers/devise.rb'
-    - 'config/environments/development.rb'
-    - 'config/environments/production.rb'
     - 'db/migrate/*'
```

Comment:
if migrate folder is not excluded in the provided config, don't exclude this as well

Reply:
Migrate folder removed from rubocop.yml exclude block.